app/routers/user.py

```diff
@@ -59,3 +61,26 @@ def get_all_user_events(session: Session, user_id: int) -> List[Event]:
         session.query(Event).join(UserEvent)
         .filter(UserEvent.user_id == user_id).all()
     )
+
+
+def disable_user(session: Session, user_id: int):
+    """this functions changes user status to disabled.
+    returns:
+    True if function worked properly 
+    False if it didn't."""
+
+    events_user_owns = list(session.query(Event).filter_by(owner_id = user_id))
+    if len(events_user_owns) == 0:
```

Comment:
Prefer a guard: `if events_user_owns: return False` and then the rest of the function, unindented.